File: Documentation/5.4/Samples/python/ClientApi/Operations/Maintenance/Configuration/GetClientConfig.py
from ravendb import GetClientConfigurationOperation, ClientConfiguration
from ravendb.documents.operations.definitions import MaintenanceOperation

from examples_base import ExampleBase


class GetClientConfig(ExampleBase):
    def setUp(self):
        super().setUp()

    def test_get_client_config(self):
        with self.embedded_server.get_document_store("GetClientConfig") as store:
            # region get_config
            # Define the get client-configuration operation
            get_client_config_op = GetClientConfigurationOperation()

            # Execute the operation by passing it to maintenance.send
            result = store.maintenance.send(get_client_config_op)

            client_configuration = result.configuration
            # endregion


class Foo:
    # region syntax_1
    class GetClientConfigurationOperation(MaintenanceOperation): ...

    # no __init__ (default)
    # endregion
    # region syntax_2
    # Executing the operation returns the following object:
    class Result:
        def __init__(self, etag: int, configuration: ClientConfiguration):
            # The configuration Etag
            self.etag = etag
            # The current client-configuration deployed on the server for the database
            self.configuration = configuration

    # endregion

File: Documentation/5.4/Samples/php/ClientApi/Operations/Maintenance/Configuration/GetClientConfig.php
<?php

namespace RavenDB\Samples\ClientApi\Operations\Maintenance\Configuration;

use RavenDB\Documents\DocumentStore;
use RavenDB\Documents\Operations\Configuration\GetClientConfigurationOperation;
use RavenDB\Documents\Operations\Configuration\GetClientConfigurationResult;

class GetClientConfig
{
    public function GetClientConfig(): void
    {
        $store = new DocumentStore();
        try {
            # region get_config
            // Define the get client-configuration operation
            $getClientConfigOp = new GetClientConfigurationOperation();

            // Execute the operation by passing it to Maintenance.Send
            /** @var GetClientConfigurationResult $result */
            $result = $store->maintenance()->send($getClientConfigOp);

            $clientConfiguration = $result->getConfiguration();
            # endregion
        } finally {
            $store->close();
        }
    }
}

/*
interface IFoo
{

    # region syntax_1
    public GetClientConfigurationOperation()
    # endregion

    # region syntax_2
    // Executing the operation returns the following object:
    class GetClientConfigurationResult implements ResultInterface
        private ?int $etag = null;
        private ?ClientConfiguration $configuration;

        // ... getters and setters
    }
    # endregion
}
*/
